migrations/db/20190117022510_easyengine_populate_hash_config_store.php

```diff
@@ -0,0 +1,140 @@
+<?php
+namespace EE\Migration;
+
+use EE;
+use EE\Model\Site;
+use EE\Model\ConfigHash;
+use EE\Utils;
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+
+class PopulateHashConfigStore extends Base {
+
+	private static $pdo;
+
+	public function __construct() {
+
+		parent::__construct();
+
+		$this->sites = Site::all();
+
+		try {
+			self::$pdo = new \PDO( 'sqlite:' . DB );
+			self::$pdo->setAttribute( \PDO::ATTR_ERRMODE, \PDO::ERRMODE_EXCEPTION );
+		} catch ( \PDOException $exception ) {
+			EE::error( $exception->getMessage() );
+		}
+
+	}
+
+	/**
+	 * Execute create table query for hash_config_store table.
+	 *
+	 * @throws EE\ExitException
+	 */
+	public function up() {
+
```

Comment:
Add skipping of migrations when it is first execution.

Reply:
Use `$this->skip_this_migration` and `$this->is_first_execution` variables which are already there in the base class you have extended.